docs/book/src/reference/watching-resources/testdata/owned-resource/controller.go

```diff
@@ -134,4 +135,4 @@ func (r *SimpleDeploymentReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&appsv1.SimpleDeployment{}).
 		Owns(&kapps.Deployment{}).
 		Complete(r)
-}
```

Comment:
The changes n the docs will make it hard we do the review and are not part of the scope 
Can you revert, please?